kratos/solving_strategies/strategies/explicit_solving_strategy.h

```diff
@@ -0,0 +1,719 @@
+     * @return mrModelPart: The model part member variable
+     */
+    inline ModelPart& GetModelPart()
+    {
+        return mrModelPart;
+    };
+
+    /**
+     * @brief Operations to get the pointer to the explicit builder and solver
+     * @return mpExplicitBuilderAndSolver: The explicit builder and solver
+     */
+    inline ExplicitBuilderAndSolverPointerType& pGetExplicitBuilderAndSolver()
+    {
+        return mpExplicitBuilderAndSolver;
+    };
+
+    /**
+     * @brief Operations to get the residual norm
+     * @return The residual norm
+     */
+    virtual double GetResidualNorm()
+    {
+        // Get the required data from the explicit builder and solver
+        const auto p_explicit_bs = pGetExplicitBuilderAndSolver();
+        const auto& r_dof_set = p_explicit_bs->GetDofSet();
+        const unsigned int dof_size = p_explicit_bs->GetEquationSystemSize();
+
+        // Calculate the explicit residual
+        p_explicit_bs->BuildRHS(GetModelPart());
+
+        // Calculate the residual norm
+        double res_norm = 0.0;
+        res_norm = IndexPartition<int>(dof_size).for_each<SumReduction<double>>(
+            [&](int i_dof) {
+                auto it_dof = r_dof_set.begin() + i_dof;
+                return it_dof->GetSolutionStepReactionValue();
+            }
+        );
+
+        return res_norm;
+    }
+
+    /**
+     * @brief Function to perform expensive checks.
+     * @details It is designed to be called ONCE to verify that the input is correct.
+     */
+    virtual int Check()
+    {
+        KRATOS_TRY
+
+        // Check if displacement var is needed
+        if (mMoveMeshFlag == true) {
+            for (const auto& r_node : GetModelPart().Nodes()) {
+                KRATOS_ERROR_IF_NOT(r_node.SolutionStepsDataHas(DISPLACEMENT)) << "ERROR:: Problem on node with Id " << r_node.Id() <<
+                "\nIt is impossible to move the mesh since the DISPLACEMENT var is not in the rModelPart. Either use (False) or add DISPLACEMENT to the list of variables" << std::endl;
+            }
+        }
+
+        // Check NODAL_AREA variable. It is needed to do the solution update
+            for (const auto& r_node : GetModelPart().Nodes()) {
+                KRATOS_ERROR_IF_NOT(r_node.SolutionStepsDataHas(NODAL_AREA)) << "ERROR:: Problem on node with Id " << r_node.Id() <<
+                "\nIt is impossible to do the solution update since the NODAL_AREA var is not in the rModelPart. Add NODAL_AREA to the list of variables" << std::endl;
+            }
+
+        // Elements check
+        for (const auto& r_elem :  GetModelPart().Elements()) {
+            const auto& r_process_info = GetModelPart().GetProcessInfo();
+            r_elem.Check(r_process_info);
+        }
+
+        // Conditions check
+        for (const auto& r_cond :  GetModelPart().Conditions()) {
+            const auto& r_process_info = GetModelPart().GetProcessInfo();
+            r_cond.Check(r_process_info);
+        }
```

Comment:
Nooooo :D
please remove see #6754

Reply:
I'd leave it until we have consensus about the scheme vs. strategy issue.